icons/148705-essential-collection/svg/003-caduceus.svg

```diff
@@ -0,0 +1,64 @@
+<?xml version="1.0" encoding="iso-8859-1"?>
```

Comment:
The essential collection is a specific set of icons. There are no more icons to add to them. We should create a separate folder for the new icons being added.